Unix/etc/omiserver.conf

```diff
@@ -16,51 +16,36 @@
 #idletimeout=TIMEOUT
 
 ##
-## trace -- enable tracing to standard output (default is 'false')
-##
-#trace=(true|false)
-
-##
-## loglevel -- set the log level of the server
+## loglevel -- set the loggiing options for MI server
+##   Valid options are: ERROR, WARNING, INFO, DEBUG, VERBOSE (debug builds only)
+##   If commented out, then default value is: WARNING
 ##
 loglevel = WARNING
 
 ##
-## <NICKNAME> -- set the value of nickname.
+## This section is for security protocol settings
+##   Including NoSSLv2,NoSSLv3,NoTLSv1_0,NoTLSv1_1,NoTLSv1_2 and NoSSLCompression.
 ##
-#prefix=PATH
-#libdir=PATH
-#bindir=PATH
-#localstatedir=PATH
-#sysconfdir=PATH
-#providerdir=PATH
-#certsdir=PATH
-#datadir=PATH
-#rundir=PATH
-#logdir=PATH
-#schemadir=PATH
-#schemafile=PATH
-#pidfile=PATH
-#logfile=PATH
-#registerdir=PATH
-#pemfile=PATH
-#keyfile=PATH
-#agentprogram=PATH
-#serverprogram=PATH
-#includedir=PATH
-#configfile=PATH
 NoSSLv2=true
 NoSSLv3=false
 NoTLSv1_0=false
 NoTLSv1_1=false
 NoTLSv1_2=false
 NoSSLCompression=false
-#NtlmCredsFile=PATH
-# For example
+
+##
+## NtlmCredsFile -- credentials file for NTLM authentication
+##
+## To enable NTLM authentication via negotiation (SPNEGO), specify the
+## location of the credentials file. This file is in the form:
+##
+##   <domain>:<username>:<password>
+##
+## For further information, please see: https://github.com/Microsoft/omi/blob/master/Unix/doc/setup-ntlm-omi.md
+##
 #NtlmCredsFile=/etc/opt/omi/.creds/ntlm
 
-## non-root feature parameters
+## non-root feature parameters, this is only for building the source code.
 ## if USER is specified within <> as the service account, 
 ##      it will be replaced with $USER when copied to output*
-#nonroot=true
 service=<USER>
```

Comment:
This USER is still used for building, I don't know if we can remove it, @palladia , how do think this part?